### DISCUSSION TITLE
How can I custom RelationManager getRecordSelect labels?

### DISCUSSION THREAD

**Discussion by bizexcel** [filamentphp/filament#12736] [Help]
### Package

Panel builder

### Package Version

v3.2.0

### How can we help you?

I'm currently having Relation manager for my resource and I'm using the AttachAction to attach my relation.
For getRecordSelect(), how can I customize their option label such as Forms\Component\Select has a method call getOptionLabelFromRecordUsing(). How can I use it for Attach record?

**Comment by sinnbeck** [comment]
Did you ever figure this out?

Edit: Figured it out :)
```php
 ->recordTitle(fn($record): string => $record->name . '(' . $record->email . ')')
